x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/field_parameters/select_inference_id.tsx

```diff
@@ -136,40 +163,48 @@ export const SelectInferenceId = ({
     };
     setOptions(Object.values(mergedOptions));
   }, [inferenceIdOptionsFromModels, defaultInferenceIds]);
-  const [isCreateInferenceApiLoading, setIsCreateInferenceApiLoading] = useState(false);
+
+  const { toasts } = useComponentTemplatesContext();
 
   const onSaveInferenceCallback = useCallback(
     async (inferenceId: string, taskType: InferenceTaskType, modelConfig: ModelConfig) => {
-      setIsCreateInferenceApiLoading(true);
+      setIsInferenceFlyoutVisible(!isInferenceFlyoutVisible);
+      const oldOptions = options;
       try {
-        await ml?.mlApi?.inferenceModels?.createInferenceEndpoint(
-          inferenceId,
-          taskType,
-          modelConfig
-        );
-        setIsInferenceFlyoutVisible(!isInferenceFlyoutVisible);
-        setIsCreateInferenceApiLoading(false);
-        setInferenceAddError(undefined);
+        const isDeployable =
+          modelConfig.service === Service.elser || modelConfig.service === Service.elasticsearch;
+        if (isDeployable) NotificationToasts({ toasts });
```

Comment:
This is never going to be an error toast so we can just show the success toast here.